### ISSUE TITLE
Support for base param when subsequent swf's are loaded by the initial swf

### ISSUE THREAD

**Issue by albertstill** [ruffle-rs/ruffle#2924]
Hi, on my [site](https://gridclub.com) most of the games request subsequent swf's after the initial screen is loaded, these live in a URL directory that is not the same as the current page and used to be specified by a nested Flash [base param](https://help.globalscape.com/help/archive/cutehtmlpro6/code/image_objet_flash.htm) `<param name="base" value="folder/to/swfs">` as a child of `<object />`. I notice that ruffle does support loading subsequent swf's but without being able to specify base I can't get a lot of the games working in the current set up. 

Is there a way for me to solve this somehow using the JS API? I've searched your JS docs and issues but can't seem to get it to work. I was hoping this would work but it didn't:

```js
player.config = {
  parameters: {
    base: '/folder/to/swfs'
  } 
};
```

Thanks for running such an awesome project!

**Comment by Herschel** [comment]
Thanks for the report. Not implemented quite yet, duplicate of #857.

**Comment by drzhupsy** [comment]
I need this functionality too! My flash files would call a subfolder to load as well.
